### ISSUE TITLE
Tech Comm weight is too big

### ISSUE THREAD

**Issue by zycon91** [CherryNetwork/polkadot#6]
The weight calculation of Tech Comm proposals is pretty big (especially when we are proposing a runtime upgrade). It is probably intentional but we still need to find a way to negate it.

**Comment by charmitro** [comment]
Is this issue fixed on the our latest [branch](https://github.com/CherryNetwork/polkadot/tree/cherry) @zycon91 ?

**Comment by zycon91** [comment]
> Is this issue fixed on the our latest [branch](https://github.com/CherryNetwork/polkadot/tree/cherry) @zycon91 ?

No. We said that we should leave it for now. Fixing the decimals of our constants made the fee a bit smaller (14-16 CHER). I can start working on it anytime.
